include/inviwo/core/util/zip.h

```diff
@@ -469,28 +469,30 @@ struct sequence {
         }
 
         iterator& operator+=(difference_type rhs) {
-            val_ += rhs * inc_;
+            val_ += static_cast<value_type>(rhs) * inc_;
             clamp();
             return *this;
         }
         iterator& operator-=(difference_type rhs) {
-            val_ -= rhs * inc_;
+            val_ -= static_cast<value_type>(rhs) * inc_;
             clamp();
             return *this;
         }
 
         difference_type operator-(const iterator& rhs) const { return ((val_ - rhs.val_) / inc_); }
         iterator operator+(difference_type rhs) const {
             auto i = *this;
-            return i += rhs;
+            return i += static_cast<value_type>(rhs);
```

Comment:
this can't be right += is defined for difference_type not value_type
same issue below...